Exiled.Events/EventArgs/Player/SpawningEventArgs.cs

```diff
@@ -51,6 +55,14 @@ public SpawningEventArgs(Player player, Vector3 position, PlayerRoleBase oldRole
         /// </remarks>
         public Vector3 Position { get; set; }
 
+        /// <summary>
+        ///     Gets or sets the <see cref="Player"/>'s spawning rotation.
+        /// </summary>
+        /// <remarks>
+        ///     Rotation will apply only for <see cref="FpcRole"/>.
+        /// </remarks>
+        public float Rotation { get; set; }
```

Comment:
```suggestion
        public float HorizontalRotation { get; set; }
```
Rename to HorizontalRotation to match the SpawnLocation APIs.